btr-web/btr-main-app/components/individual-person/effective-dates/startEndGroup.vue

```diff
@@ -17,8 +17,8 @@
           v-if="isEndDateVisible"
           :class="{'bg-blue-500': highlightRow}"
           :name="name + '.endDate'"
-          :initial-date="dateStringToDate(dates.endDate || '') || undefined"
-          :min-date="dates.startDate"
+          :initial-date="dates.endDate ? dateStringToDate(dates.endDate) as Date: undefined"
+          :min-date="dates.startDate ? dateStringToDate(dates.startDate) as Date: undefined"
           :max-date="new Date()"
```

Comment:
I think this could be changed in the start date component (line 9) to: `dates.endDate ? dateStringToDate(dates.endDate) : new Date()`

Reply:
good point, will do that